spec/unit/inconfig_spec.rb

```diff
@@ -13,7 +13,7 @@
     end
 
     it "validate_configuration" do
-      SgtnClient.load("./spec/config/sgtnclient-invalidate.yml", "test", './sgtnclient_config.log')
```

Comment:
Why comment this line?

Reply:
This loads an invalid config file, which gets consumed by next set of tests.